hack/jenkins/e2e.sh

```diff
@@ -217,6 +217,9 @@ if [[ "${E2E_UP,,}" == "true" ]]; then
         # other.
         export KUBE_SKIP_UPDATE=y
         sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update -q" || true
+        sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update preview -q" || true
+        sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update alpha -q" || true
```

Comment:
hm, maybe some sort of weird race with another `gcloud update` call or something?

Running `gcloud components list` from the Jenkins master VM shows that the preview, alpha, and beta components are all installed and up-to-date.